apps/explorer/src/app/components/oracle-table/index.tsx

```diff
@@ -0,0 +1,159 @@
+import compact from 'lodash/compact';
```

Comment:
Note: This component is not new, it has just been pulled out of `Oracles /home / index.tsx` so it can be reused